### ISSUE TITLE
Loss value is not decrease during training

### ISSUE THREAD

**Issue by futorio** [CNChTu/Diffusion-SVC#8]
Hello! I have been training a model using 1204 files, with each file taking between 5 and 25 seconds to process. The total duration of the training process has been 14031 seconds. However, despite training for approximately 3 hours, I have noticed that the loss function is not decreasing. I would appreciate any insights into why this might be happening and any suggestions on how to resolve the issue. Thank you!

configs/duffision.yaml file:

```
data:
  block_size: 512
  cnhubertsoft_gate: 10
  duration: 2
  encoder: whisper-ppg
  encoder_hop_size: 320
  encoder_out_channels: 1024
  encoder_sample_rate: 16000
  extensions:
  - wav
  sampling_rate: 44100
  training_files: filelists/train.txt
  unit_interpolate_mode: nearest
  validation_files: filelists/val.txt
device: cuda
env:
  expdir: logs/44k/diffusion
  gpu_id: 0
infer:
  method: dpm-solver
  speedup: 10
model:
  n_chans: 512
  n_hidden: 256
  n_layers: 20
  n_spk: 1
  type: Diffusion
  use_pitch_aug: true
spk:
  speaker_all: 0
train:
  amp_dtype: fp32
  batch_size: 384
  cache_all_data: true
  cache_device: cpu
  cache_fp16: true
  decay_step: 100000
  epochs: 100000
  gamma: 0.5
  interval_force_save: 10000
  interval_log: 10
  interval_val: 2000
  lr: 0.00008
  num_workers: 2
  save_opt: false
  weight_decay: 0.01
vocoder:
  ckpt: pretrain/nsf_hifigan/model
  type: nsf-hifigan
```
![train loss](https://github.com/CNChTu/Diffusion-SVC/assets/33608846/f9b84e42-07c5-43cd-9232-d439812543bb)

![validation](https://github.com/CNChTu/Diffusion-SVC/assets/33608846/3f8fe875-f02e-4449-92c8-3249a7c2e912)

**Comment by yxlllc** [comment]
This is normal for the training of the diffusion model. The guiding significance of the loss is not so great. As the training progresses, even if the loss does not decrease, it may still improve the quality of the synthesis. Of course, be careful of the occurrence of overfitting.

**Comment by futorio** [comment]
Thank you for answer!

**Comment by yxlllc** [comment]
Also I noticed that your batchsize is as high as 384, which might be a bit extreme. If the batchsize is too large, there is a risk of falling into a local optimum, which also leads to poor convergence

**Comment by futorio** [comment]
> Also I noticed that your batchsize is as high as 384, which might be a bit extreme. If the batchsize is too large, there is a risk of falling into a local optimum, which also leads to poor convergence

So i have 24 gb vram on RTX 4090. Would i Can I increase the learning rate to compensate for the effect of batch size and at the same time make efficient use of my graphics card's memory?
